dapp/src/components/buySell/SellWidget.js

```diff
@@ -170,13 +170,26 @@ const SellWidget = ({
     )
     coinData.ousd = ousdToSell
 
+    const [rebaseThreshold] = await Promise.all([
+      vaultContract.rebaseThreshold(),
+    ])
```

Comment:
Could the above be simplified as: 
```
const rebaseThreshold = await vaultContract.rebaseThreshold()
```

Reply:
Ah, yes 😇 I meant to come back and actually read that line but got in a hurry.